src/views/Upload/Upload.js

```diff
@@ -1,7 +1,90 @@
-import React from "react";
+import React, { useState } from 'react';
+import Button from '@mui/material/Button'
+import Box from '@mui/material/Box';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import IconButton from '@mui/material/IconButton';
+import DeleteIcon from '@mui/icons-material/Delete';
+import pageOption from '../../utils/pageOption';
+import UnderlineHeader from '../../components/UnderlineHeader/UnderlineHeader';
+import ContinueButton from '../../components/ContinueButton/ContinueButton';
+import PropTypes from 'prop-types';
+import './Upload.css';
 
-const Upload = () => {
-    return <h1>Upload</h1>;
+const Upload = ({ setPage, uploadedFiles, setUploadedFiles }) => {
+  const [files, setFiles] = useState(uploadedFiles);
+
+  const continueAction = () => {
+    if (!files) {
```

Comment:
This should check if files is empty also print error message `!files || files.length === 0`